cpg_workflows/stages/seqr_loader_long_read/long_read_snps_indels_annotation.py

```diff
@@ -0,0 +1,428 @@
+                    stage=SiteOnlyVCFs,
+                    target=multicohort,
+                    key='siteonly_part_pattern',
+                ).format(idx=idx),
+            )
+            for idx in range(scatter_count)
+        ]
+
+        jobs = add_vep_jobs(
+            get_batch(),
+            input_siteonly_vcf_path=inputs.as_path(multicohort, stage=SiteOnlyVCFs, key='siteonly'),
+            input_siteonly_vcf_part_paths=input_siteonly_vcf_part_paths,
+            out_path=outputs['ht'],
+            tmp_prefix=self.tmp_prefix / 'tmp',
+            job_attrs=self.get_job_attrs(),
+            scatter_count=scatter_count,
+        )
+        return self.make_outputs(multicohort, outputs, jobs)
+
+
+@stage(required_stages=[MergeLongReadSNPsIndels, VepLRS])
+class AnnotateCohortLRSNPsIndels(MultiCohortStage):
+    """
+    First step to transform annotated SNPs Indels callset data into a seqr ready format
+    """
+
+    def expected_outputs(self, multicohort: MultiCohort) -> dict[str, Path]:
+        """
+        Expected to write a matrix table.
+        """
+        return {'mt': self.prefix / 'cohort_snps_indels.mt'}
+
+    def queue_jobs(self, multicohort: MultiCohort, inputs: StageInput) -> StageOutput | None:
+        """
+        queue job(s) to rearrange the annotations prior to Seqr transformation
+        """
+        outputs = self.expected_outputs(multicohort)
+
+        vcf_path = inputs.as_path(target=multicohort, stage=MergeLongReadSNPsIndels, key='vcf')
+        vep_ht_path = inputs.as_path(target=multicohort, stage=VepLRS, key='ht')
+
+        job = annotate_cohort_jobs_snps_indels(
+            vcf_path=vcf_path,
+            out_mt_path=outputs['mt'],
+            vep_ht_path=vep_ht_path,
+            checkpoint_prefix=self.tmp_prefix / 'checkpoints',
+            job_attrs=self.get_job_attrs(multicohort),
+        )
+
+        return self.make_outputs(multicohort, data=outputs, jobs=job)
+
+
+@stage(required_stages=[AnnotateCohortLRSNPsIndels], analysis_type='custom', analysis_keys=['mt'])
+class AnnotateDatasetLRSNPsIndels(DatasetStage):
+    """
+    Subset the MT to be this Dataset only
+    """
+
+    def expected_outputs(self, dataset: Dataset) -> dict:
+        """
+        Expected to generate a matrix table
+        """
+
+        return {
+            'mt': (dataset.prefix() / 'mt' / f'LongReadSNPsIndels-{get_workflow().output_version}-{dataset.name}.mt'),
+        }
+
+    def queue_jobs(self, dataset: Dataset, inputs: StageInput) -> StageOutput | None:
+        """
+        Subsets the whole MT to this cohort only
+        Then brings a range of genotype data into row annotations
+
+        Args:
+            dataset (Dataset): SGIDs specific to this dataset/project
+            inputs ():
+        """
+        assert dataset.cohort
+        assert dataset.cohort.multicohort
+        mt_path = inputs.as_path(target=dataset.cohort.multicohort, stage=AnnotateCohortLRSNPsIndels, key='mt')
```

Comment:
```suggestion
        mt_path = inputs.as_path(target=get_multicohort(), stage=AnnotateCohortLRSNPsIndels, key='mt')
```

possibly coupled with an import of `cpg_workflows.workflow.get_multicohort`. This was my jank before I realised there was a cleaner way to get to the MC object